src/nav/MainNavBarHome.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Not sure if this needs to be a different component.
Now `MainNavBar` is almost an empty component.